src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=" "
+      method="post"
+      data-qa="big"
+    >
+      <div class="search-wrap search-wrap-big">
+        <label>
+          <input
+            type="text"
+            placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
Add data-qa attribute with the value 'keypress' to the small search bar input as well.